public/styles/credits.css

```diff
@@ -73,22 +70,57 @@ main {
   .video {
     aspect-ratio: 16 / 9;
     height: auto;
-    width: 900px;
+    width: 1000px;
     max-width: 100%;
+    display: block;
+    margin: 1rem auto 0 auto;
+  }
+
+  .bigups {
+    cursor: pointer;
```

Comment:
I don't think this is necessary. It's not clickable.

Reply:
It's not necessary, but there is a `title` attribute on Avros' name with a shoutout, so I just added it so maybe a few people might notice.